Hindawi Publishers.js

```diff
@@ -80,7 +89,27 @@ function doWeb(doc,url)
 				// use article ID as a page (seems to be the last part of URL/DOI)
 				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
 			}
-			item.extra = "";
+			//remove duplicate metadata			
```

Comment:
Correct, just the number not the label. That's because the EM import works
with ZU.xpathText
On Mon, Jan 4, 2016 at 8:43 AM, Aurimas Vinckevicius <
notifications@github.com> wrote:

> In Hindawi Publishers.js
> <https://github.com/zotero/translators/pull/972#discussion_r48736252>:
>
> > @@ -80,7 +89,27 @@ function doWeb(doc,url)
> >  				// use article ID as a page (seems to be the last part of URL/DOI)
> >  				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
> >  			}
> > -			item.extra = "";
> > +			//remove duplicate metadata			
>
> Oh, I see. What about the PMID? The PMID label is not duplicated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/translators/pull/972/files#r48736252>.
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com